src/pages/_document.tsx

```diff
@@ -25,4 +55,4 @@ class MyDocument extends Document {
   }
 }
 
-export default MyDocument
+export default Document
```

Comment:
I decided to rename `MyDocument` to `Document` and the previous `Document` to `NextDocument`. I think it makes things more clear.